### ISSUE TITLE
Use share-permissions webdav property

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#1796]
Use new webdav property "share-permissions". 
This property contains an integer which is the max share-permissions. This will allow you to display to proper share permissions etc.

Scenario:
- user A shares a folder to user B with no delete permission

@rullzer

**Comment by rullzer** [comment]
Yep. This is just a webdav property so you know how to share the file. 

Let me explain the scenario:

> user A shares a folder to user B with no delete permission

userB receives the share from userA. Now userB can always remove the incoming share. However, when userB tries to share the file to userC it is impossible to share with delete permissions.
